java/change-notes/2021-04-06-ssrf-query.md

```diff
@@ -0,0 +1,2 @@
+lgtm,codescanning
+* The server-side request forgery (SSRF) query has been promoted from experimental to the main query pack. Its results will now appear by default.
```

Comment:
Should we credit the original author in the change note? There was some discussion about whether or not to do this in https://github.com/github/codeql-release-notes/pull/10, but I don't know if that reached a conclusion.
I also think that we should include the query id, we tend to reference queries in change notes by `name (id)`.

Reply:
The C/C++ team seems to do that:
https://github.com/github/codeql/pull/5256/files#diff-6947ef4ef4cef28d3a7a76c1aec757c8695b621922b33c4a5c00ca443c2558cd